hosts/runner/github-runner.nix

```diff
@@ -1,12 +1,7 @@
-{ lib, pkgs, hostName, ... }:
+{ lib, pkgs, hostName, runnerGroup, runners, ... }:
 
 let
-  runnersNames = [
-    "${hostName}-aa"
-    "${hostName}-bb"
-    "${hostName}-cc"
-    "${hostName}-dd"
-  ];
+  runnersNames = map (name: "${hostName}-${name}") runners;
```

Comment:
Needed to allow multiple "sizes" of runners supporting different numbers of parallel builds.

Alternative: currying, but I think adding this "global" argument is ok.